toolbox/siestabarriers/SiestaBarriersBase.py

```diff
@@ -0,0 +1,318 @@
+# -*- coding: utf-8 -*-
+########################################################################################
+# Copyright (c), The SiestaBarriers authors. All rights reserved.                       #
+#                                                                                      #
+# SiestaBarriers is hosted on GitHub at https://github.com/.................. #
+# For further information on the license, see the LICENSE.txt file                     #
+########################################################################################
+
+from __future__ import absolute_import
+
+#from SiestaBarriers.Utils.utils_siesta import print_siesta_fdf,read_siesta_fdf
+
+from .Utils.utils_siesta import print_siesta_fdf,read_siesta_fdf
+#from Utils.utils_general import AtomIndex
+#from SiestaBarriers.Utils.utils_siesta import print_siesta_fdf
+from .Utils.utils_siesta import print_siesta_fdf
+import sys
+import inspect
+pacakge_dir= inspect.getabsfile(print_siesta_fdf).split("utils_siesta.py")[0]+"flos/"
+#pacakge_dir= inspect.getabsfile(print_siesta_fdf).split("neb_base.py")[0]+"flos/"
+
+
+__author__ = "Arsalan Akhatar"
+__copyright__ = "Copyright 2021, SIESTA Group"
+__version__ = "0.1"
+__maintainer__ = "Arsalan Akhtar"
+__email__ = "arsalan_akhtar@outlook.com," #+ \" miguel.pruneda@icn2.cat "
+__status__ = "Development"
+__date__ = "Janurary 30, 2021"
```

Comment:
Date is probably too much. Is it really needed?

Reply:
jajajaj it was fast copy paste from the source ...